.gitignore

```diff
@@ -10,4 +10,5 @@ log/*
 .tool-versions
 newrelic_agent.log
 .vscode
+.idea
```

Comment:
Seems I am the first one who uses Idea :)

Reply:
Hmmm weird. That was in there before 😕